e2e/testsuite/testsuite.go

```diff
@@ -177,29 +177,29 @@ func (s *E2ETestSuite) GetPathName(idx int64) string {
 }
 
 // generatePath generates the path name using the test suites name
-func (s *E2ETestSuite) generatePath(ctx context.Context, relayer ibc.Relayer) string {
+func (s *E2ETestSuite) generatePath(ctx context.Context, ibcrelayer ibc.Relayer) string {
```

Comment:
nit: relayer seems like a term highly specific to ibc, not sure about benefit of adding it to name here